Elevel/Application/Features/TopicCommands/DeleteTopicCommand.cs

```diff
@@ -34,9 +34,10 @@ public async Task<Response> Handle(Request request, CancellationToken cancelatio
                     return null;
                 }
                 _context.Topics.Remove(topic);
+                topic.Deleted = true;
```

Comment:
you should or Hard Delete or Soft Delete approach

Reply:
the Soft Delete approach has been chosen for this issue